.github/workflows/python-package.yml

```diff
@@ -10,13 +10,29 @@ on:
     branches: [ "*" ]
 
 jobs:
-  build:
+  prepare-cache:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Cache KRISTINsnid.csv.zip
+        id: cache-kristin
+        uses: actions/cache@v4
+        with:
+          path: src/islenska/resources/KRISTINsnid.csv.zip
+          key: kristin-csv-zip-v1
+      - name: Download KRISTINsnid.csv.zip
+        if: steps.cache-kristin.outputs.cache-hit != 'true'
+        run: |
+          mkdir -p src/islenska/resources
+          wget -q -O src/islenska/resources/KRISTINsnid.csv.zip 'https://bin.arnastofnun.is/django/api/nidurhal/?file=KRISTINsnid.csv.zip'
```

Comment:
I'm not that familiar with the CI build process. Does it run the actual build process for the DAWG? Under what circumstances does this then generate a new version of the content of BinPackage, i.e. based on a fresh download of KRISTINsnid.csv.zip? Usually the production of the DAWG needs to be supervised since errors can show up in the build process, as has been the case a few times in the past.